schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, <% if(!!viewEncapsulation) { %>, ViewEncapsulation<% }%><% if(changeDetection !== 'Default') { %>, ChangeDetectionStrategy<% }%> } from '@angular/core';
+import { BreakpointObserver, Breakpoints, BreakpointState } from '@angular/cdk/layout';
+import { Observable } from 'rxjs/Observable';
+
+@Component({
+  selector: '<%= selector %>',<% if(inlineTemplate) { %>
```

Comment:
So I guess schematics doesn't have a way to insert contents of another file here? Do they have a way to add comments that will be stripped out of the generated file? If so it would be good to add some comments for people maintaining this file that they need to update in 2 places

Reply:
Ya, its very painful. I'm thinking about making a util to handle this automatically.